Ports/epsilon/package.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=epsilon
+version=15.5.0
+files="https://github.com/numworks/epsilon/archive/refs/tags/${version}.tar.gz ${port}-${version}.tar.gz 38c3b6baaf00863bbd179bce5e9cc42bbdbd0cd485b5bf3bbf4473383591bf83"
+auth_type=sha256
+makeopts="PLATFORM=simulator TARGET=serenity SERENITY_INSTALL_ROOT=${SERENITY_INSTALL_ROOT}"
+depends="SDL2 libpng libjpeg"
+
+
+install() {
+    run mkdir -p ${SERENITY_INSTALL_ROOT}/opt/
+    run cp output/release/simulator/serenity/epsilon.elf ${SERENITY_INSTALL_ROOT}/opt/epsilon
```

Comment:
Why not install to `/usr/local/bin/`, like other ports?

Reply:
Not [all ports](https://github.com/SerenityOS/serenity/blob/5c89305fc6fce3aeb50725341b95caa8e8da7e5c/Ports/Another-World/package.sh#L20) are installed in `/usr/loca/bin` (and I didn't even see one of them yet).
But I'm going to change that yep